### PR TITLE
Released import/export from tech-preview state.

### PR DESCRIPTION
**Pull request by ggainey**

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]
This needs the fixes stanza

**Review by daviddavis — APPROVED** [review]

**Comment by ipanova** [comment]
moved to draft, we might want to keep it in tech-preview for 3.11

File: CHANGES/8146.feature [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Removed 'tech preview' label from the import/export documentation.
+
+PulpImport/PulpExport is now a fully-supported part of pulp3.
```
Comment (daviddavis) [inline CHANGES/8146.feature:3]: This should be a doc file.

**Comment by ggainey** [comment]
Can we mark this as Draft? There may be an API change needed based on katello's feedback

**Comment by ipanova** [comment]
@ggainey any updates here?

**Comment by ggainey** [comment]
> @ggainey any updates here?

We're on-hold pending what happens in ansible-land RE https://pulp.plan.io/issues/8204 - if we need to do this work, it *might* affect the API. Also, while we've addressed all the API-affecting requests from katello, will request a final "ack" from them before moving fwd on this PR/issue.

**Comment by bmbouter** [comment]
I'd like us to get more data on usage of this feature before removing the tech preview. Also I think we should close these PRs that have been open for so long without a clear plan on when the tech preview labels will be removed.

**Pull request closed without merging**